core/Utils.vala

```diff
@@ -636,7 +636,13 @@ namespace Maya.Util {
     }
 
     public void style_calendar_color (Gtk.Widget widget, string color, bool background = false) {
-        string style = ".cal_color { %s: %s }";
+        string style = """
+            .cal_color {
+                %s: %s;
+                border-radius: 999px;
+                padding: 0 0.75em;
```

Comment:
It would probably be better to either use `min-height` and `min-width` here or `height_request` and `width_request` in Gtk.

Something else to consider is using a Gtk.Grid instead of a Gtk.Label since Label will reserve extra space for text inside of it